uw-frame-components/portal/features/services.js

```diff
@@ -236,7 +236,10 @@ define(['angular'], function(angular) {
               //check dates
               var today = Date.parse(new Date());
               var startDate = Date.parse(new Date(feature.goLiveYear, feature.goLiveMonth, feature.goLiveDay));
-              var expirationDate = feature.buckyAnnouncement.endDate;
+              var isoDate = feature.buckyAnnouncement.endDate;
+              var temp = new Date(isoDate);
+              var expirationDate = temp.getTime();
```

Comment:
This is fine. I'd be tempted to avoid the `temp` and do `var expirationDate = new Date(isoDate).getTime();`.